packages/cxl-lumo-styles/scss/themes/vaadin-horizontal-layout.scss

```diff
@@ -5,3 +5,17 @@
     color: var(--lumo-primary-color);
   }
 }
+
+:host([theme~="cxl-marketing-hero-details"]) {
+  font-size: var(--lumo-font-size-s);
+
+  ::slotted(span) {
+    display: inline-flex;
+    align-items: center; /* In Firefox, for some weird reason, things look more center *without* this property. */
+    margin-right: var(--lumo-space-s);
```

Comment:
Maybe another rule with `:not(:last-child)`?

Reply:
And looking at story markup, flex `gap` is the correct move over margin.

Which means icon should be grouped inside span.

Reply:
> Which means icon should be grouped inside span.

How do I style icon inside a `span`? I couldn't make this work:

```css
::slotted(span) {
    [icon^="vaadin:"] {...}
}
```

Reply:
You're missing basics about shadow DOM styling https://stackoverflow.com/questions/61626493/slotted-css-selector-for-nested-children-in-shadowdom-slot

This requires an element global style, **but** since this safe area seems contained to a specific documented pixel value, what if we created a new theme `vaadin-icon[theme="lumo"]` with safe area?

It would ensure nothing currently working fine breaks, and we can opt in individual icons to see how it all works out, on the way to maybe making a single global rule for all vaadin-icon in the future.

Reply:
> You're missing basics about shadow DOM styling

Yes, this is true. I can say I was not working with shadow DOM besides some learning and experiments.

> what if we created a new theme vaadin-icon[theme="lumo"] with safe area?

🚢